### DISCUSSION TITLE
Does EfficientAD resize input size to 256 x 256?

### DISCUSSION THREAD

**Discussion by stevensagaert** [open-edge-platform/anomalib#2238] [Q&A]
Hi,
I've trained an EfficientAD model on a custom dataset. I'm using the small model. I have not specified any image size (it's all in python code , not the cli). Normally I'd expect the model to keep the original resolution but I'm not sure that what happens during training/testing. Is this true or do the images automatically get scaled back to 256 * 256?

**Comment by abc-125** [comment]
Hi, yes, [unless you specify custom transform, images are resized to 256*256](https://github.com/openvinotoolkit/anomalib/blob/1a041d137d4d58cc9cdce563519c3f376357beee/src/anomalib/models/image/efficient_ad/lightning_model.py#L326C36-L326C46). Here is the link on [custom transforms](https://anomalib.readthedocs.io/en/latest/markdown/guides/how_to/data/transforms.html). Hope this helps.

**Reply by WestwardWinds** [reply]
If you are making a custom dataset for training using `Folder()` and specify `image_size=(x, y)` there, that will prevent the images from being resized during training without including a specific custom transform, correct? Or do you still need a image transformation? 

I ask because I am also doing EfficentAD training right now and all my source images are 416x416 and my intention is to run the trained model on that image size once it is finished training.
